src/Model/Widget.php

```diff
@@ -76,18 +63,16 @@ class Widget extends DataObject
      */
     private static $description = "Description of what this widget does.";
 
-    /**
-     * @var array
-     */
-    private static $summary_fields = array(
+    private static $summary_fields = [
         'CMSTitle' => 'Title'
-    );
+    ];
 
-    /**
-     * @var string
-     */
     private static $table_name = 'Widget';
 
+    private static $extensions = [
+        Versioned::class . '.versioned',
```

Comment:
Remove the ` . '.versioned'` as discussed with @tractorcow - this is the wrong kind of versioning for this application